django/cantusdb_project/requirements.txt

```diff
@@ -1,6 +1,22 @@
+appdirs==1.4.4
 asgiref==3.2.7
-Django==3.0.7
+attrs==19.3.0
+black==19.10b0
+click==7.1.2
+Django==3.0.6
```

Comment:
Was the downgrade intentional?

Reply:
Nope, GitHub had updated it but when I did ``pip freeze`` it went back. I'll fix it, good catch